lib/gui/app/pages/main/templates/main.tpl.html

```diff
@@ -66,11 +66,11 @@
               'text-disabled': main.shouldDriveStepBeDisabled()
             }">
             <span class="drive-step step-name"
-              uib-tooltip="{{ main.selection.getDrive().description }} ({{ main.selection.getDrive().displayName }})">
+              uib-tooltip="{{ main.selection.getCurrentDrive().description }} ({{ main.selection.getCurrentDrive().displayName }})">
               <!-- middleEllipses errors on undefined, therefore fallback to empty string -->
-              {{ (main.selection.getDrive().description || "") | middleEllipses:11 }}
+              {{ (main.selection.getCurrentDrive().description || "") | middleEllipses:11 }}
```

Comment:
In `selection-state.js` there's a bunch of "helper methods" like `getImageName` and `getImageSize`. Would it also make sense for there to be "helper methods" like `getCurrentDriveDescription()` and `getCurrentDriveSize()` ?